src/video/emscripten/SDL_emscriptenvideo.c

```diff
@@ -307,6 +309,24 @@ static void Emscripten_SetWindowSize(_THIS, SDL_Window * window)
     }
 }
 
+
+static void
+Emscripten_GetWindowDrawableSize(_THIS, SDL_Window * window, int *w, int *h)
+{
+    SDL_WindowData *data;
+    if (window->driverdata) {
+        data = (SDL_WindowData *) window->driverdata;
+
+        if (w) {
+            *w = window->w * data->pixel_ratio;
```

Comment:
You may need some rounding here.

Reply:
Ah yeah, good call... right now this is doing the same thing as `Emscripten_GLES_GetDrawableSize` (in fact it might make sense to route `Emscripten_GLES_GetDrawableSize` to the same function as it's basically duplicate code right now).